packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -479,24 +481,29 @@ const disableVmHighAvailability = async (xapi: Xapi, vm: Vm) => {
 // Attributes on created VM snapshots:
 //
 // - `other_config`:
-//    - `xo:backup:datetime` = snapshot.snapshot_time (allow sorting replicated VMs)
 //    - `xo:backup:deltaChainLength` = n (number of delta copies/replicated since a full)
 //    - `xo:backup:exported` = 'true' (added at the end of the backup)
+//
+// Attributes on created VMs and created snapshots:
+//
+// - `other_config`:
+//    - `xo:backup:datetime`: format is UTC %Y%m%dT%H:%M:%SZ
+//       - from snapshots: snapshot.snapshot_time
+//       - with offline backup: formatDateTime(Date.now())
 //    - `xo:backup:job` = job.id
 //    - `xo:backup:schedule` = schedule.id
 //    - `xo:backup:vm` = vm.uuid
 //
 // Attributes of created VMs:
 //
-// - all snapshots attributes (see above)
 // - `name_label`: `${original name} - ${job name} - (${safeDateFormat(backup timestamp)})`
 // - tag:
 //    - copy in delta mode: `Continuous Replication`
 //    - copy in full mode: `Disaster Recovery`
 //    - imported from backup: `restored from backup`
 // - `blocked_operations.start`: message
 // - for copies/replications only, added after complete transfer
-//    - `other_config[xo:backup:sr]` = sr.uuid
+//    - `other_config[xo:backup:sr]` = sr.$id
```

Comment:
Why this change?

Reply:
Because we use the SR id not the SR uuid.

Reply:
ATM, the `$id`is the UUID, but it can change in the future and we don't want that.

I think we should use the UUID explicitely.

Reply:
I'll do it in other PR because it'll produce unrelated changes.

Reply:
Ok, I'm doing it directly on `master` and you'll rebase on top of this.

Reply:
Thank you :+1: